quail/processes/wps_climdexInput_raw.py

```diff
@@ -154,10 +162,20 @@ def __init__(self):
     def generate_dates(
         self, request, filename, obj_name, date_fields, date_format, cal
     ):
-        load_rdata_to_python(filename, obj_name)
-        return robjects.r(
-            f"as.PCICt(do.call(paste, {obj_name}[,{date_fields}]), format='{date_format}', cal='{cal}')"
-        )
+        load_rda(filename, obj_name)
+        try:
+            return robjects.r(
+                f"as.PCICt(do.call(paste, {obj_name}[,{date_fields}]), format='{date_format}', cal='{cal}')"
+            )
+        except RRuntimeError as e:
+            raise ProcessError(msg="Error generating dates")
```

Comment:
If we aren't going to list that this was a `RRuntimeError` we could catch a general exception and raise it as a `ProcessError`.